src/data/vapor-thermal-conductivity-constants.ts

```diff
@@ -0,0 +1,778 @@
+        C2: -0.0036161,
+        C3: 5.6641E-6,
+        C4: -2.8451e-9,
+        minimumTemperature: 436.42,
+        maximumTemperature: 706.95,
+        thermalConductivityAtMinimumTemperature: 0.05147,
+        thermalConductivityAtMaximumTemperature: 0.05647,
+      },
+
+    ButyricAcidExtended: {
+        name: "Butyric acid",
+        formula: "C4H8O2",
+        CAS: "107-92-6",
+        molecularWeight: 88.1051,
+        C1: 9.2069e-8,
+        C2: 2.0312,
+                minimumTemperature: 706.95,
+        maximumTemperature: 1000,
+        thermalConductivityAtMinimumTemperature: 0.05647,
+        thermalConductivityAtMaximumTemperature: 0.11421,
+      },
+*/
+  Butyronitrile: {
+    name: "Butyronitrile",
+    formula: "C4H7N",
+    CAS: "109-74-0",
+    molecularWeight: 69.1051,
+    C1: 1.3751e-6,
+    C2: 1.5786,
+    minimumTemperature: 390.74,
+    maximumTemperature: 1000,
+    thermalConductivityAtMinimumTemperature: 0.01698,
+    thermalConductivityAtMaximumTemperature: 0.07484
+  },
+
+  CarbonDioxide: {
+    name: "Carbon dioxide",
+    formula: "CO2",
+    CAS: "124-38-9",
+    molecularWeight: 44.0095,
+    C1: 3.69,
+    C2: -0.3838,
+    C3: 964,
+    C4: 1860000,
+    minimumTemperature: 194.67,
+    maximumTemperature: 1500,
+    thermalConductivityAtMinimumTemperature: 0.00887,
+    thermalConductivityAtMaximumTemperature: 0.09025
+  }
+};
+
+export function calculateDIPPRVaporThermalConductivity(compound: string, temperature: number): number {
+  const vaporThermalConductivityProps: DIPPRVaporThermalConductivityProps | undefined =
+    DIPPRVaporThermalConductivityDictionary[compound];
+  let thermalConductivity = NaN;
+  if (
+    vaporThermalConductivityProps !== undefined &&
+    typeof vaporThermalConductivityProps.minimumTemperature === "number" &&
+    typeof vaporThermalConductivityProps.maximumTemperature === "number" &&
+    temperature >= vaporThermalConductivityProps.minimumTemperature &&
+    temperature <= vaporThermalConductivityProps.maximumTemperature
+  ) {
+    thermalConductivity =
+      (vaporThermalConductivityProps.C1 * Math.pow(temperature, vaporThermalConductivityProps.C2)) /
+      (1 +
+        (Number.isFinite(vaporThermalConductivityProps.C3)
+          ? (vaporThermalConductivityProps.C3 as number) / temperature
+          : 0) +
+        (Number.isFinite(vaporThermalConductivityProps.C4)
+          ? (vaporThermalConductivityProps.C4 as number) / Math.pow(temperature, 2)
+          : 0));
+  }
+  return thermalConductivity;
+}
+/* function to calculate thermal coductivity of acetic acid, butyric acid, formic acid, heptanoic acid, octanoic acid, pentanoic acid, propionic acid,
+
+export function calculateDIPPRThermalConductivityException(compound: string, temperature: number): number {
+  const vaporThermalConductivityProps: DIPPRVaporThermalConductivityProps | undefined =
+    DIPPRVaporThermalConductivityDictionary[compound];
+  let thermalConductivity = NaN;
+  if (
```

Comment:
- [ ] Uncomment this function
- [ ] Call this function inside `calculateDIPPRVaporThermalConductivity`, such that it gets fired only for those "special" compounds.
- [ ] Uncomment all the special compounds